Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -129,14 +150,28 @@ - (void)initializeWithParameters:(id<MAAdapterInitializationParameters>)paramete
                 ALVungleIntializationStatus = MAAdapterInitializationStatusInitializedFailure;
                 NSString *errorString = [NSString stringWithFormat: @"%ld:%@", (long) error.code, error.localizedDescription];
                 
-                completionHandler(ALVungleIntializationStatus, errorString);
+                @synchronized(completionHandlers)
+                {
+                    for ( void (^handler)(MAAdapterInitializationStatus, NSString *_Nullable) in completionHandlers )
+                    {
+                        handler(ALVungleIntializationStatus, errorString);
```

Comment:
The errorString is not consumed in any of the handler so it isn't needed here because each of the ad format has its own unique log about failing.. Maybe just log it after L151.